user_assigned_identity/outputs.tf

```diff
@@ -14,3 +14,23 @@ output "tenant_id" {
   value = azurerm_user_assigned_identity.identity.tenant_id
 }
 
+output "identity" {
```

Comment:
Refactor this so each attribute is their own `output`.
E.g:
`module.user_assigned_identity["min"].role_assignment` as opposed to `module.user_assigned_identity["min"].identity.role_assignment`